semeio/jobs/rft/zonemap.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+
+class ZoneMap:
+    def __init__(self, zones_at_k_value=None):
+        self._zones_at_k_value = zones_at_k_value or {}
+        self._k_values_at_zone = {}
+
+        for k_value, zone_names in self._zones_at_k_value.items():
+            for zone_name in zone_names:
+                if not zone_name in self._k_values_at_zone:
+                    self._k_values_at_zone[zone_name] = []
+
+                self._k_values_at_zone[zone_name].append(k_value)
+
+    def __contains__(self, item):
+        if isinstance(item, int):
+            return item in self._zones_at_k_value
+        elif isinstance(item, str):
```

Comment:
Unnecessary `elif` after `return`

Reply:
You mean it could be `if` instead? Is it _that_ bad to use `elif`?

Reply:
I'm not entirely sure - for me it doesn't feel right, and for pylint it doesn't either..... `Unnecessary "elif" after "continue" (no-else-continue)`

Don't think it's that bad - feel free to just leave it as is :+1: